site/_apps/hnpwa-wc.txt

```diff
@@ -0,0 +1,26 @@
+---
+weight: 69
+title:  'HNPWA Webcomponents'
+github-title: 'tenosiswono/hnpwa-wc'
+libraries: 
+    - name: 'Webcomponentsjs'
+    - name: 'Redux'
+module-bundling: 'Webpack'
+service-worker: 'Application Shell with Workbox'
+perfomance-patterns: 'Server push, preload'
+server-side-rendering: 'No'
+api: 'hnpwa.com/api'
+hosting: 'Firebase'
+authors: 
+    - name: 'Teno'
+lighthouse: '91/100'
+interactive-em: '3.1s'
+interactive-faster-3g: '2.4s'
```

Comment:
With newer traces these numbers seem to be a bit slower (especially EM/Slow 3G). Could you please update the numbers here? 

Regular 3G: https://www.webpagetest.org/result/180629_2S_24c6bb9620c25086dd87346c1f759428/
Slow 3G: https://www.webpagetest.org/result/180629_7A_11ffcb6fe41678ac561bd9daae06f8a1/